earthaccess/store.py

```diff
@@ -533,7 +533,7 @@ def _get_granules(
         provider = granules[0]["meta"]["provider-id"]
         endpoint = self._own_s3_credentials(granules[0]["umm"]["RelatedUrls"])
         cloud_hosted = granules[0].cloud_hosted
-        access = "direc" if (cloud_hosted and self.running_in_aws) else "external"
+        access = "direct" if (cloud_hosted and self.running_in_aws) else "external"
```

Comment:
Thinking out loud: I think it'd be really useful to add type safety here. We could type `access` as Enum or as a union of literal string types.